### ISSUE TITLE
Going to Lessons > Add Lesson results in an error

### ISSUE THREAD

**Issue by brianhogg** [gocodebox/lifterlms#2636]
### Reproduction Steps

1. Go to Lessons, then click the Add Lesson button

### Expected Behavior

You are able to add a new lesson

### Actual Behavior

Fatal error trying to check for the course drip settings, when there's no course for the lesson yet.

### Error Messages / Logs

```
PHP Fatal error:  Uncaught Error: Call to a member function get() on null in /Users/brianhogg/laravel/lifter2/wp-content/plugins/lifterlms/includes/admin/post-types/meta-boxes/class.llms.meta.box.lesson.php:164
Stack trace:
#0 /Users/brianhogg/laravel/lifter2/wp-content/plugins/lifterlms/includes/abstracts/abstract.llms.admin.metabox.php(349): LLMS_Meta_Box_Lesson->get_fields()
#1 /Users/brianhogg/laravel/lifter2/wp-content/plugins/lifterlms/includes/abstracts/abstract.llms.admin.metabox.php(278): LLMS_Admin_Metabox->process_fields()
#2 /Users/brianhogg/laravel/lifter2/wp-admin/includes/template.php(1456): LLMS_Admin_Metabox->output(Object(WP_Post), Array)
#3 /Users/brianhogg/laravel/lifter2/wp-admin/includes/post.php(2382): do_meta_boxes(Object(WP_Screen), 'normal', Object(WP_Post))
#4 /Users/brianhogg/laravel/lifter2/wp-admin/edit-form-blocks.php(316): the_block_editor_meta_boxes()
#5 /Users/brianhogg/laravel/lifter2/wp-admin/post-new.php(72): require('/Users/brianhog...')
#6 /Users/brianhogg/.composer/vendor/laravel/valet/server.php(250): require('/Users/brianhog...')
#7 {main}
  thrown in /Users/brianhogg/laravel/lifter2/wp-content/plugins/lifterlms/includes/admin/post-types/meta-boxes/class.llms.meta.box.lesson.php on line 164
```

This issue has be recreated:
+ [ ] Locally
+ [ ] On a staging site
+ [ ] On a production website
+ [ ] With only LifterLMS and a default theme

**Comment by chrisbadgett** [comment]
This error is also happening when cloning a lesson. It creates a cloned lesson as a draft, but clicking edit returns a white screen. 

![CleanShot 2024-05-02 at 11 27 45@2x](https://github.com/gocodebox/lifterlms/assets/12163552/38d1cc54-ea88-4b3d-945b-63bab4ae48fa)

**Comment by brianhogg** [comment]
Clone and Add Lesson should both be fixed in the latest release (7.6.1)
